### PR TITLE
samples: basic: blinky: add mbox backend for eGPIO

### PR DESCRIPTION
**Pull request by jaz1-nordic**

### CODE REVIEW

File: samples/basic/blinky/sample.yaml [hunk 1]
```diff
@@ -10,3 +10,30 @@ tests:
...
     integration_platforms:
       - frdm_k64f
+
+  sample.basic.blinky.egpio_icmsg:
+    tags:
+      - LED
+      - gpio
+    filter: dt_enabled_alias_with_parent_compat("led0", "gpio-leds")
+    depends_on: gpio
+    harness: led
```
Comment (nordic-piks) [inline samples/basic/blinky/sample.yaml:20]: There is no such harness `led`, it is dummy one.
Please change to `console` and add expected regex from sample.

Comment (nordic-piks) [reply samples/basic/blinky/sample.yaml:20]: You can do sth similar to this:
https://github.com/zephyrproject-rtos/zephyr/pull/72730/files

**Review by nordic-piks — CHANGES_REQUESTED** [review]
Please allow verification via console logs for blinky sample.

File: samples/basic/blinky/sample.yaml [hunk 2]
```diff
@@ -10,3 +10,45 @@ tests:
...
+      - SB_CONFIG_EGPIO_FLPR_APPLICATION=y
+
+  sample.basic.blinky.egpio_mbox:
+    tags:
+      - LED
+      - gpio
+    filter: dt_enabled_alias_with_parent_compat("led0", "gpio-leds")
+    depends_on: gpio
+    integration_platforms:
+      - nrf54l15pdk/nrf54l15/cpuapp
```
Comment (nordic-piks) [inline samples/basic/blinky/sample.yaml:42]: Can you add DK target too?

File: samples/basic/blinky/sample.yaml [hunk 3]
```diff
@@ -10,3 +10,45 @@ tests:
...
       - frdm_k64f
+
+  sample.basic.blinky.egpio_icmsg:
+    tags:
+      - LED
+      - gpio
+    filter: dt_enabled_alias_with_parent_compat("led0", "gpio-leds")
+    depends_on: gpio
+    integration_platforms:
+      - nrf54l15pdk/nrf54l15/cpuapp
```
Comment (nordic-piks) [inline samples/basic/blinky/sample.yaml:21]: Can you add DK target too?

**Review by nordic-piks — APPROVED** [review]

File: samples/basic/blinky/sysbuild.conf [hunk 4]
```diff
@@ -0,0 +1 @@
+SB_CONFIG_PARTITION_MANAGER=n
```
Comment (masz-nordic) [inline samples/basic/blinky/sysbuild.conf:1]: This could be moved to `sample.yaml` to limit impact.

**Review by magp-nordic — APPROVED** [review]

**Review by masz-nordic — APPROVED** [review]

**Review by magp-nordic — APPROVED** [review]

**Review by nordic-segl — APPROVED** [review]

**Review by carlescufi — DISMISSED** [review]
why is this a noup instead of a fromlist?

**Review by gmarull — COMMENTED** [review]
I don't think we should be carrying these sort of patches downstream. They do not seem to follow best practices, and will likely keep increasing upmerge times. Why isn't this developed directly upstream?

**Comment by carlescufi** [comment]
Please use the alt-root instead: https://github.com/nrfconnect/sdk-nrf/tree/main/scripts/twister/alt/zephyr
Check with @PerMac for additional info about the twister alt-root we have in Zephyr.

**Pull request merged**